### ISSUE TITLE
Update Kubelet Configuration link to refer to stable v1 release in `Configuring a cgroup driver` page

### ISSUE THREAD

**Issue by sca-ai** [kubernetes/website#41587]
The introductory paragraph of the english documentation for "Configuring a cgroup driver" discusses differences between kubelet's systemd and cgroupfs driver usage in the context of kubeadm. Here is an outdated reference to an older beta version of the "Kubelet Configuration" documentation (kubelet-config.v1beta1). Shouldn't we point to the current documentation for the stable v1 release (kubelet-config.v1)?

**Comment by dipesh-rawat** [comment]
**_Page related to issue:_** https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/configure-cgroup-driver
/language en

**Comment by dipesh-rawat** [comment]
/retitle Update Kubelet Configuration link to refer to stable v1 release in `Configuring a cgroup driver` page

**Comment by neolit123** [comment]
the page in question shows v1, which exists as an api but is not a full component config api.
https://kubernetes.io/docs/reference/config-api/kubelet-config.v1/
https://github.com/kubernetes/kubelet/blob/master/config/v1/types.go

we need to understand what to do with this.
@tengqm


/sig node

**Comment by neolit123** [comment]
@ndixita 
i see you worked on the kubelet v1, is this a wip api that should not be used by users yet? it has missing fields / structs from v1beta1.

https://github.com/kubernetes/kubernetes/commits/master/staging/src/k8s.io/kubelet/config/v1

**Comment by sca-ai** [comment]
The PR for this issue was closed (#41588).
I shouldn't have [commented on the PR](https://github.com/kubernetes/website/pull/41588#issuecomment-1587764530). This issue is the right place for it. Can anyone shed some light on it, please?

**Comment by neolit123** [comment]
https://github.com/kubernetes/website/pull/41588#issuecomment-1588291086

TL;DR v1 is still WIP and should not be linked to.

/close
